tls/s2n_resume.c

```diff
@@ -261,30 +299,330 @@ int s2n_connection_get_session(struct s2n_connection *conn, uint8_t *session, si
 
     uint32_t len = s2n_connection_get_session_length(conn);
 
+    if (!len) {
+        return len;
```

Comment:
nit: return 0 and len == 0 seems easier to read (:

Reply:
Okay.